.ct/lintconf.yaml

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
Why not just use the defaults of `ct`?

Reply:
I don't remember the exact reason why I had to add this, probably something with the added stuff broke the lint.
If you prefer, we can remove this and see what fails and see if that can be solved without this file.